plan.tex

```diff
@@ -1,5 +1,9 @@
 \section{Planning and team(s) fro DP0} \label{sec:plan}
 
+Planning epics have been (and are) being created in the \href{https://jira.lsstcorp.org/secure/Dashboard.jspa?selectPageId=15608}{PREOPS Jira} project.
+On the dashboard you can see links to the  tickets tagged DP0.1 and DP0.2.
+
+We will have regular (every other week for now) DP0 meetings see \url{https://confluence.lsstcorp.org/display/LSSTOps/Data+Production+Meetings}.
```

Comment:
There has to be punctuation between "meetings" and "see" (maybe use parentheses around "see url").